crates/typst-library/src/math/matrix.rs

```diff
@@ -270,6 +270,15 @@ pub struct CasesElem {
     #[default(Delimiter::Brace)]
     pub delim: Delimiter,
 
+    /// Whether the direction fo cases should be reversed.
```

Comment:
```suggestion
    /// Whether the direction of cases should be reversed.
```

Reply:
I should really start proofreading my PRs.